### ISSUE TITLE
Cannot save email settings (This form should not contain extra fields)

### ISSUE THREAD

**Issue by CherryDT** [SolidInvoice/SolidInvoice#1016]
**Describe the bug**
Trying to update email settings (because the From address is wrong) results in an error "This form should not contain extra fields"

**To Reproduce**
1. Go to email settings
2. Click "Save"

**Expected behavior**
Saving

**Screenshots**
![image](https://github.com/SolidInvoice/SolidInvoice/assets/6010463/2475ddb8-b84e-4abd-ad28-2fc415f627f8)

**Desktop (please complete the following information):**
- OS: Windows
- Browser Chrome
- Version 123.0.6312.123

**Comment by CherryDT** [comment]
OK it worked after I set the mail provider here, the error is pretty misleading though, as there were no _extra_ fields but rather _missing_ fields, apparently!

Plus, I didn't change anything other than the email address (the mail provider was empty here by default, even though I set `MAILED_DSN` in `.env`, which I had to do because otherwise I couldn't log in, see #1014).

Also, even when there is an error submitting the form, it shouldn't clear the fields like it did in this screenshot!
